WORKSPACE

```diff
@@ -81,6 +66,8 @@ npm_install(
     name = "npm",
     package_json = "//:package.json",
     package_lock_json = "//:package-lock.json",
+    symlink_node_modules = True,
```

Comment:
When we symlink the node_modules can we integrate them in the docker image?

Reply:
We can't integrate them in a clean way either way. I have checked both options and True is the recommended default value there